packages/api/controllers/hackathon_team_members_controller.go

```diff
@@ -61,13 +62,26 @@ func CreateHackathonMember(c *fiber.Ctx) error {
 		SponsorShare:          member.SponsorShare,
 		NewsLetter:            member.NewsLetter}
 
+	cursor, err := teamMembersCollection.Find(
+		ctx,
+		bson.D{{"email", newHackathonTeamMember.Email}},
+	)
+	var results []models.TeamMember
+	if err = cursor.All(ctx, &results); err != nil {
```

Comment:
Why didn't you implement this? https://www.mongodb.com/docs/manual/core/index-unique/

Reply:
I tried with a search index but I wasn't happy with the results. If I have to I would change it.

Reply:
Can you elaborate? What were the results?

Reply:
This is [part ](https://www.mongodb.com/docs/drivers/go/current/fundamentals/crud/read-operations/text/ )of the Mongo DB documentation I looked at. And it was getting all documents in the DB. Now I am going to try and make it with unique index.